conf/trusts.routes

```diff
@@ -49,7 +49,9 @@ PUT         /:identifier/trust-details/inter-vivos
 PUT         /:identifier/trust-details/efrbs-start-date                                     controllers.transformations.trustdetails.TrustDetailsTransformationController.setEfrbsStartDate(identifier: String)
 PUT         /:identifier/trust-details/residential-status                                   controllers.transformations.trustdetails.TrustDetailsTransformationController.setResidentialStatus(identifier: String)
 
-DELETE      /:identifier/transforms/migrate-to-taxable                                      controllers.TrustVariationsController.removeTaxableMigrationTransforms(identifier: String)
+GET         /:identifier/transforms/migrate-to-taxable                                      controllers.TaxableMigrationController.getTaxableMigrationFlag(identifier: String)
+POST        /:identifier/transforms/migrate-to-taxable                                      controllers.TaxableMigrationController.setTaxableMigrationFlag(identifier: String)
```

Comment:
As this connects to a new mongo repository and isn't a _tranform_ internall can we think of an alternative endpoint?